cmd/posix_test.go

```diff
@@ -73,6 +73,31 @@ func TestPosixGetDiskInfo(t *testing.T) {
 	}
 }
 
+func TestPosixIsDirEmpty(t *testing.T) {
```

Comment:
Can you add a successful test case? an empty directory and isEmpty() returning true.